docs/docs/getting-started/first-steps.md

```diff
@@ -13,19 +13,51 @@ If you encounter any problem with the following steps, make sure to use the [lat
 
 ## Create a cluster
 
-1. Create the configuration file and IAM resources for your selected cloud provider
+1. Create the [configuration file](../workflows/config.md)
 
-    First, you need to create a [configuration file](../workflows/config.md) and an [IAM configuration](../workflows/config.md#creating-an-iam-configuration). The easiest way to do this is the following CLI command:
+    First, you need to create your [configuration file](../workflows/config.md). This can be easily done with the following command.
```

Comment:
This can be shortened to
```md
1. Create the [configuration file](../workflows/config.md) for your cloud provider
```
The other two sentences don't contain any new information. (MS style guide: be brief)